### ISSUE TITLE
subvolumegroup quota metrics?

### ISSUE THREAD

**Issue by jhoblitt** [galexrt/extended-ceph-exporter#14]
I am interested in scraping cephfs subvolumegroup quota metrics (`bytes_quota` & `bytes_used`). Is a cephfs collector a feature that would potentially be accepted into this exporter or is the intent for it to be focused on rgw?

**Comment by galexrt** [comment]
Yes, this would be something I gladly accept as a feature. "Extended" RGW bucket metrics was simply the first thing implemented back then ;-)
